__tests__/hooks/useEtherspotHistory.test.js

```diff
@@ -0,0 +1,63 @@
+import { renderHook, waitFor } from '@testing-library/react';
+
+// hooks
+import { useEtherspotHistory, EtherspotUi } from '../../src';
+
+describe('useEtherspotHistory()', () => {
+  it('getAccountTransactions() returns account history', async () => {
+    const wrapper = ({ children }) => (
+      <EtherspotUi provider={null}>
+        {children}
+      </EtherspotUi>
+    );
+
+    const { result, rerender } = renderHook(({ chainId }) => useEtherspotHistory(chainId), {
+      initialProps: { chainId: 1 },
+      wrapper,
+    });
+
+    // wait for history to be fetched for chain ID 1
+    await waitFor(() => expect(result.current).not.toBeNull());
+
+    const accountTransactionsMainnet = await result.current.getAccountTransactions();
+    expect(accountTransactionsMainnet.length).toEqual(2);
+    expect(accountTransactionsMainnet[0].hash).toEqual('0x1');
+    expect(accountTransactionsMainnet[0].value).toEqual('100000000000000');
+    expect(accountTransactionsMainnet[1].hash).toEqual('0x2');
+
+    // rerender with different chain ID 137
+    rerender({ chainId: 137 });
+
+    const accountTransactionsPolygon = await result.current.getAccountTransactions();
+    expect(accountTransactionsPolygon.length).toEqual(0);
+  });
+  it('getAccountTransaction() returns transaction by existing hash', async () => {
+    const wrapper = ({ children }) => (
+      <EtherspotUi provider={null}>
+        {children}
+      </EtherspotUi>
```

Comment:
As above

Reply:
Oops sorry, was not updated to master, fixed!